cgi-bin/DW/VirtualGiftTransaction.pm

```diff
@@ -71,7 +71,7 @@ sub save {
 
     my $u       = LJ::want_user( $opts{user} ) or return;
     my $id      = LJ::alloc_user_counter( $u, 'V' ) or return;
-    my $secs    = $opts{time} || localtime;
```

Comment:
Was trying to insert 'Sat Jan 23...' string into the int field expecting an epoch time.